src/WordPress/PluginRoutes.php

```diff
@@ -2,34 +2,20 @@
 
 namespace CF\WordPress;
 
-class PluginRoutes
+class PluginRoutes extends \CF\API\PluginRoutes
 {
-    public static $routes = array(
-        'account' => array(
-            'class' => 'CF\WordPress\PluginActions',
-            'methods' => array(
-                'POST' => array(
-                    'function' => 'loginWordPress',
-                ),
-            ),
-        ),
+    /**
+     * @param $routeList
+     *
+     * @return mixed
+     */
+    public static function getRoutes($routeList)
+    {
+        foreach ($routeList as $routePath => $route) {
+            $route['class'] = '\CF\WordPress\PluginActions';
+            $routeList[$routePath] = $route;
```

Comment:
Ah nevermind, its cause `$route` is scoped locally to the function.  This is fine.